src/main/webapp/app/tasks/task-search/task-search.component.scss

```diff
@@ -0,0 +1,8 @@
+.mat-form-field{
+    width: 100%;
+}
+
+.mt{
+    margin-top: 1rem;
+}
+::ng-deep  .mat-focused .mat-form-field-underline .mat-form-field-ripple{ background: greenyellow }
```

Comment:
ng-deep is deprecated.

Reply:
Acknowledged, Will look for work around.